manifest.konnector

```diff
@@ -12,6 +12,47 @@
     "online_services"
   ],
   "fields": {
+    "provider": {
+      "type": "dropdown",
+      "default": {
```

Comment:
This attribute does not work with the last cozy-home. Please remove it.

Reply:
You still have some code to handle default value in the connector itself

Reply:
@doubleface You're right, the default value is already handled in the konnector. I removed the "default" attribute.